app/components/EditProfileForm.tsx

```diff
@@ -49,11 +54,21 @@ export default function EditProfileForm({userData}) {
 
     const data = await res.json();
     console.log(data)
+    const status = res.status
+
+    if (status == 200) {
+      setErrorMessage("")
```

Comment:
No need to to `setErrorMessage` here - it will make React re-render the component. Just redirect.